Cognigy/WebchatWidget
# Feature/28694 default preview

This PR is adding a new setting to WebchatWidget enableDefaultPreview, that allows choosing preference to a new field for rendering _defaultPreview.
This setting should not affect any previous behaviour when disabled. but when enabled should take prefer _defaultPreview for rendering with message-renderer, messenger plugin and adaptive cards plugins. 

There were extensive tests written to make sure that we do not miss the most logical and used cases. Spec file related is priorites.spec.ts

How to test:
1. Run Automated tests with `npm run test`
To test manually:
1. Run `npm run dev`
2.  in the index.html add the new setting flag
3. send a webchat.sendMessage() with different payloads to check
